data/GunjanShakya.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "Gunjan Shakya",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "A final year student with keen interest in Data Structures and Algorithms",
+  "avatar": "https://avatars.githubusercontent.com/u/93340830?s=400&u=d618cf71e10817941deb633d63f5b49af17ae12a&v=4",
```

```suggestion
  "avatar": "hhttps://github.com/GunjanShakya.png",
```
Please use this icon as the avatar